File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/screen/user_detail/UserView.java
package ru.avystavkin.githubgists.screen.user_detail;

import java.util.List;

import androidx.annotation.NonNull;
import ru.avystavkin.githubgists.models.local.Gist;
import ru.avystavkin.githubgists.screen.general.ErrorView;

public interface UserView extends ErrorView {
    void showGists(@NonNull List<Gist> gists);
}

File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/screen/user_detail/UserView.kt
package ru.avystavkin.githubgists.screen.user_detail

import ru.avystavkin.githubgists.models.database.Gist
import ru.avystavkin.githubgists.screen.general.ErrorView

interface UserView : ErrorView {
    fun showGists(gists: List<Gist>)
}
